Apps/WebClient/src/ClientApp/src/models/configData.ts

```diff
@@ -63,6 +65,58 @@ export interface WebClientConfiguration {
     offlineMode?: OfflineModeConfiguration;
 }
 
+// Configuration data to be used by the Health Gateway Webclient.
+export interface FeatureToggleConfiguration {
+    homepage: HomepageSettings;
+    waitingQueue: WaitingQueueSettings;
+    notificationCentre: NotificationCentreSettings;
+    timeline: TimelineSettings;
+    dataset: DatasetSettings[];
+    covid19: Covid19Settings;
+    dependents: DependentsSettings;
+}
+
+export interface HomepageSettings {
+    showFederalProofOfVaccination: boolean;
+}
+
+export interface WaitingQueueSettings {
+    enabled: boolean;
+}
+
+export interface NotificationCentreSettings {
+    enabled: boolean;
+}
+
+export interface TimelineSettings {
+    comment: boolean;
+}
+
+export interface DatasetSettings {
+    name: string;
+    enabled: boolean;
+}
+
+export interface Covid19Settings {
+    pcrTestEnabled: boolean;
+    publicCovid19: PublicCovid19;
+    proofOfVaccination: ProofOfVaccination;
+}
+
+export interface PublicCovid19 {
+    enableTestResults: boolean;
+    showFederalProofOfVaccination: boolean;
+}
+
+export interface ProofOfVaccination {
+    exportPdf: boolean;
+}
```

Comment:
The names of these interfaces could be appended with "Settings", as could their back-end counterparts.  I'm mostly concerned about the possibility of them showing up as auto-complete suggestions.